core/src/main/java/io/questdb/cairo/DefaultCairoConfiguration.java

```diff
@@ -423,6 +423,11 @@ public boolean isSqlParallelFilterEnabled() {
         return true;
     }
 
+    @Override
+    public boolean isSqlParallelFilterPreTouchEnabled() {
+        return false;
```

Comment:
is there a reason to default this to false?

Reply:
Well, if the data is in the page cache and/or the filtered columns are subsequent, pre-touch would be wasting CPU time. So that's why I propose to keep it disabled, at least for now.

Reply:
would it be possible to measure impact on CPU if we keep pre-touch enabled?

Reply:
It highly depends on the exact scenario, but I'll measure some of them on my machine. Will post an update here.

Reply:
Here is what I tested:
```sql
CREATE TABLE IF NOT EXISTS tab (
  id SYMBOL capacity 1000000 nocache,
  sh SHORT,
  f0 float,
  f1 float,
  f2 float,
  f3 float,
  f4 float,
  f5 float,
  f6 float,
  f7 float,
  f8 float,
  f9 float,
  f10 float,
  f11 float,
  f12 float,
  ts TIMESTAMP,
  created_at TIMESTAMP
) timestamp(created_at) partition by DAY;

insert into tab
select
  concat('id', x % 35151),
  rnd_short(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  rnd_float(),
  cast(x * 2500 as timestamp),
  cast(x * 2500 as timestamp)
from long_sequence(218047822);

select *
from tab
where id = 'id50';

select *
from tab
where id in ('id0', 'id101', 'id202', 'id303', 'id404', 'id505', 'id606', 'id707', 'id808', 'id909');
```

The first query returns 6.2K rows, while the second one returns 62K.

There is a jitter in the query execution time with both enabled and disabled pre-touch, so comparison is not straight-forward, but I didn't notice any additional slowness when pre-touch is enabled.

Reply:
Considering this, we could optimistically enable pre-touch by default. In this case we should keep in mind that it's possible that this feature may slow things down in niche scenarios, like a low-end machine with small RAM capacity where the worker threads would be contending for the page cache.

Reply:
thank you @puzpuzpuz !

Reply:
Sounds like a deal. I'm going to change the default to `true`.

Reply:
WDYT?

Reply:
I’m thinking - we will do this explicitly for cloud?

Reply:
PR is already approved and ready to go

Reply:
> I’m thinking - we will do this explicitly for cloud?

We could override this prop for Cloud instances.

Reply:
> PR is already approved and ready to go

No problem with that, I can enable the feature by default in a separate PR if we decide to do so.